ui_testing/testcases/basic_tests/test06_orders.py

```diff
@@ -558,15 +564,13 @@ def test013_update_order_number(self):
         self.assertEqual(new_orders_count,
                          records_in_analysis_after_update_count)
 
-    # will continue with us
-    def test014_update_order_material_type(self, save):
+    @parameterized.expand(['save_btn', 'cancel'])
+    def test014_update_first_order_material_type(self, save):
```

Comment:
why you update This test case in this PR, while travis file contains only LIMS-3406

Reply:
@ghada-37 all of those comments because this pr not merged from the master branch at first time so I merged it and pull all the changed fro the master branch

Reply:
Done